.kokoro/tests/run_tests.sh

```diff
@@ -146,6 +146,7 @@ test_prog="${PROJECT_ROOT}/.kokoro/tests/run_single_test.sh"
 
 btlr_args=(
     "run"
+    "--max-cmd-duration=30m"
```

Comment:
What happens when it timeouts? I presume the test will fail. Am I right?

Reply:
Yes (specifically, it sends a sigterm to the command and which will end the python process and give a non-zero exit code). 

The error reported will be "signal: killed" in the summary statement for the cmd.